### PR TITLE
feat(PatchHelper): Paste Full Patch

### PR DESCRIPTION
**Pull request by MrDiamondDog**
image

Pasting your patch into the `Full Patch` field will auto-fill all the other fields

### CODE REVIEW

File: src/components/CheckedTextInput.tsx [hunk 1]
```diff
@@ -55,6 +55,10 @@ export function CheckedTextInput({ value: initialValue, onChange, validate }: Te
         }
     }
 
+    useEffect(() => {
```
Comment (Vendicated) [inline src/components/CheckedTextInput.tsx:58]: ?

Comment (MrDiamondDog) [reply src/components/CheckedTextInput.tsx:58]: I forgot why I did this, I think something didn't work without it

Comment (MrDiamondDog) [reply src/components/CheckedTextInput.tsx:58]: This is necessary so that when I use setMatch it will update the checked text input, otherwise it doesn't

**Comment by Vendicated** [comment]
thanks

**Pull request merged**